molecule/default/molecule.yml

```diff
@@ -94,6 +101,35 @@ provisioner:
       # space and the slash.  This option converts those characters to
       # underscores when creating the group name.
       force_valid_group_names: always
+  inventory:
+    host_vars:
+      debian9:
```

Comment:
Ah, I just noticed on this one and may have missed on other PRs. Should this be `debian9_systemd`?

Reply:
This definitely causes an issue with Python 2 vs 3, as the [cisagov/ansible-role-cyhy-runner](https://github.com/cisagov/ansible-role-cyhy-runner) PR failed testing until I updated that value.

Reply:
See commit 90344a1.

Reply:
Sorry I only just noticed 😞

Reply:
Good catch @mcdonnnj - better now than later! 👍